packages/workflow/src/NodeHelpers.ts

```diff
@@ -642,21 +678,39 @@ export function getNodeParameters(nodePropertiesArray: INodeProperties[], nodeVa
 			}
 
 			// Iterate over all collections
-			for (const itemName of Object.keys(propertyValues || {})) {
-				if (nodeProperties.typeOptions !== undefined && nodeProperties.typeOptions.multipleValues === true) {
+			for (const itemName of Object.keys(propertyValues ?? {})) {
```

Comment:
Maybe not ideal to express skipping the loop as looping over the keys of an empty object.

Reply:
agree, but that should be part of a code review not of the style 😝